README.md

```diff
@@ -46,7 +45,7 @@ between three core domains:
 * **Education**  - demonstrating that open play rather than rigid structures increases motivation and engagement in the classroom
 
 Again, finding a balance working to the best benefits of all these
-domains is the objective.
+domains are the objective.
```

Comment:
@tiagomoraismorgado88 the original word here is actually correct. (This becomes clearer if you put the last few words near the front of the sentence):
'Again, _the objective is_ finding a balance working to the best benefits of all these domains.'